lib/VeresOne.js

```diff
@@ -132,15 +137,14 @@ class VeresOne {
    * Fetches a DID Document for a given DID. First checks the ledger, and if
    * not found, also checks local DID storage (for pairwise DIDs).
    *
-   * @param did {string} URI of a DID, either registered on a ledger, or
-   *   unregistered (pairwise cryptonym DID).
-   *
-   * @param [autoObserve=false] {boolean} Start tracking changes to the DID Doc
-   *   (to generate a diff patch later).
-   *
+   * @param {Object} params - Contains a did and an authObserve property.
```

Comment:
We've already established a pattern of using `options` instead of `params`, please continue using that pattern.

Reply:
the issue here is that param has options which results in options.options.accelator.